pkg/system/reconciler.go

```diff
@@ -98,6 +98,8 @@ type Reconciler struct {
 	IBMCosBucketCreds         *corev1.Secret
 	DefaultBackingStore       *nbv1.BackingStore
 	DefaultBucketClass        *nbv1.BucketClass
+	DefaultNamespaceStore     *nbv1.NamespaceStore
+	DefaultPVC                *corev1.PersistentVolumeClaim
```

Comment:
maybe the name should be DefaultNsfsPvc because default pvc is a bit vague in the operator context